examples/player/simid_player.html

```diff
@@ -23,15 +23,30 @@
     }
   </style>
   <script>
+    //if ad is non-linear allow user to insert specific dimensions
+    function enableDisable() {
```

Comment:
If I'm understanding this correctly, I don't think we should do this in the sample code. We should just allow the ad to change dimensions (if possible).

We are adding complexity we might not need.

Reply:
when we originally spoke you mentioned only allowing specified sizing for non-linear ads. Would you rather allow sizing for linear as well?